vizro-core/docs/pages/explanation/schema.md

```diff
@@ -0,0 +1,156 @@
+                "x": "sepal_length",
+                "y": "petal_width"
+              },
+              "type": "graph"
+            },
+            {
+              "figure": {
+                "_target_": "histogram",
+                "color": "species",
+                "data_frame": "iris",
+                "x": "sepal_width"
+              },
+              "type": "graph"
+            }
+          ],
+          "controls": [
+            {
+              "column": "species",
+              "type": "filter"
+            }
+          ],
+          "title": "My first dashboard"
+        }
+      ]
+    }
+    ```
+
+=== "maps to this Dashboard"
+    [![Dashboard]][dashboard]
+
+!!! note
+    The Vizro schema is still incomplete. This means that it does not yet define everything that we consider to be core and supported functionality of Vizro. The most prominent example of an omission is the [`CapturedCallable`][vizro.models.types.CapturedCallable], which are the objects you insert into many models such as `vm.Graph`, `vm.Table` or `vm.Action`, often into the `figure` argument.
+
+    This does not mean that it cannot be configured in JSON. In fact, in the configuration above, we have used the following JSON:
+
+    ```json
+    {
+      "figure": {
+        "_target_": "histogram",
+        "color": "species",
+        "data_frame": "iris",
+        "x": "sepal_width"
+      },
+      "type": "graph"
+    }
+    ```
+
+    It means that `figure` does not appear in the schema so, as such, it is not clearly defined as part of the Vizro framework, although it should be considered as such. We are working to complete the Vizro schema in the future.
+
+## The role of Pydantic
+
+The Vizro framework is powered by [Pydantic](https://docs.pydantic.dev/latest/), which is the most widely used data validation library for Python. Pydantic acts as the glue that connects Vizro's JSON schema to its actual implementation - a React frontend served by a Flask backend, facilitated by Dash.
+
+One core advantage of Pydantic is that it emits a [JSON schema](https://blog.postman.com/what-is-json-schema) so that our models, such as `vm.Page` or `vm.Filter`, can easily be translated into a well-defined JSON schema. Beyond this capability, it also has support for custom validation beyond the functionality of the JSON schema.
+
+In our toy example from above we could define that, in the array of strings, consecutive elements have to have more letters than their predecessor. This is a fairly random example, but it illustrates the power of custom validation for anyone using the Vizro framework. Most of the time when the user "misconfigures", the error message is clear and concise.
+
+## The grammar of dashboards
+
+How does this all come together? One of the long-term goals of Vizro is to define a so-called **grammar of dashboards**. This means that we complete the Vizro schema and define a unified, implementation independent language to configure dashboards.
```

Comment:
This section should definitely link to other grammars:
* grammar of graphics
* grammar of interactive graphics
* plotly JSON schema

These are ultimately what inspires the term "grammar of dashboards" and roughly what we seek to emulate, even if we can never fully realise it to the same extent as them.